### ISSUE TITLE
UninitializedPropertyAccessException: lateinit property featuredImageView has not been initialized

### ISSUE THREAD

**Issue by loremattei** [wordpress-mobile/WordPress-Android#13196]
**Sentry Url**: https://sentry.io/share/issue/ca318ff6adde4e07bebdfecda260ca54/
**User Count**: 6
**Count**: 33
**First Release**: org.wordpress.android@16.0-rc-1+936
**First Seen**: 2020-10-20T11:33:09.386000Z
**Last Seen**: 2020-10-21T06:49:44Z
**24 Hours**: 34
**30 Days**: 34

```
UninitializedPropertyAccessException: lateinit property featuredImageView has not been initialized
     at org.wordpress.android.ui.reader.ReaderPostDetailFragment.access$getFeaturedImageView$p(ReaderPostDetailFragment.kt:140)
     at org.wordpress.android.ui.reader.ReaderPostDetailFragment$ShowPostTask.onPostExecute(ReaderPostDetailFragment.kt:1087)
     at org.wordpress.android.ui.reader.ReaderPostDetailFragment$ShowPostTask.onPostExecute(ReaderPostDetailFragment.kt:981)
...
(2 additional frame(s) were not displayed)
```

**Comment by loremattei** [comment]
This has not a big impact right now, but since it's new in the latest beta I think it's worth monitoring and some investigation.

**Comment by develric** [comment]
Hi @ashiagr , I see you self-assigned so I'm assuming you are having a look into this already, correct? I set a low pri for now but not removing the Required Triage since as per Lore comment above it's under monitoring and we can change priority in case numbers rise.

**Comment by ashiagr** [comment]
Hi @develric, 

I was able to reproduce this issue. Linked PR fixes it. Let me know if you can review it else will ask @zwarm as she has more context.

**Comment by develric** [comment]
Hey @ashiagr thanks for wrangling a fix! I agree @zwarm can have more context so I'm happy with her reviewing the PR, of course let me know in case I can help 🙇 .

**Comment by ashiagr** [comment]
Fixed in #13202
